test/unknown-attributes-filter.spec.coffee

```diff
@@ -0,0 +1,126 @@
+debug = require('debug')('spec:unknown-attributes-filter')
+_ = require 'underscore'
+_.mixin require 'underscore-mixins'
+{UnknownAttributesFilter} = require '../lib'
+Promise = require 'bluebird'
+{ExtendedLogger} = require 'sphere-node-utils'
+package_json = require '../package.json'
+
+sampleAttributeNameList = [
+  'attributeName1'
+,
+  'attributeName2'
+,
+  'attributeName3'
+]
+
+sampleAttributeList = [
```

Comment:
It is a list of attributes that might have some unknown attributes along with the defined attributes. So cannot actually call them `unexpectedAttributes`. It is supposed to be a normal list of attributes.